src/App.tsx

```diff
@@ -1,61 +1,44 @@
+// #region imports
+import { useState } from 'react';
 import './App.scss';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { TodoForm } from './components/TodoForm';
+import { getUserById } from './services/user';
+// #endregion
```

Comment:
Remove comments. Try it write obvious code that explain all we need without comments. Fix all cases